dart/tool/java/generate_java.dart

```diff
@@ -950,7 +951,8 @@ class TypeField extends Member {
           if (refName.endsWith('Ref'))
             refName = "@" + refName.substring(0, refName.length - 3);
           w.addLine(
-              'if (elem.get("type").getAsString().equals("${refName}")) return new ${t.name}(elem);');
+              'if (elem.get("type").getAsString().equals("${refName}")) return new ${t
```

Comment:
I did a double take on this line because it is a single line string spanning multiple lines. It is correct because the line break appears inside the interpolation block, but perhaps split this string in two before the return?

Reply:
Yeah - really hard to read. I'll add braces and reformat :)